app/src/main/java/net/gsantner/opoc/ui/FilesystemViewerAdapter.java

```diff
@@ -601,7 +601,9 @@ private boolean canWrite(File file) {
     public boolean accept(File dir, String filename) {
         final File f = new File(dir, filename);
         final boolean filterYes = f.isDirectory() || _dopt.fileOverallFilter == null || _dopt.fileOverallFilter.apply(f);
-        final boolean dotYes = _dopt.showDotFiles || !filename.startsWith(".");
+        final boolean dotYes = _dopt.showDotFiles || !filename.startsWith(".") &&
```

Comment:
maybe add another (brace around the new conditions. otherwise - it's working/OK for you this way? 

If so, this is ready for merge

Reply:
Finally I add a function to make it more clear. How do you think?